poetry.lock

```diff
@@ -223,6 +223,15 @@ version = "5.1"
 [package.extras]
 toml = ["toml"]
 
+[[package]]
+category = "main"
+description = "Manage Python errors with ease"
+marker = "python_version >= \"3.6\" and python_version < \"4.0\""
+name = "crashtest"
```

Comment:
I can see crashtest on pypi but not on github, is this expected?

Reply:
It's not open source yet, I need to write a proper README for the project first. It should be ready tomorrow.

Reply:
It's now open source: https://github.com/sdispater/crashtest